apps/bridge-dapp/src/pages/PageBridge.tsx

```diff
@@ -46,6 +48,8 @@ import {
 } from '../hooks';
 import { downloadNotes } from '../utils';
 
+const STATS_DEV_URL = 'https://www.stats-dev.webb.tools/';
```

Comment:
We shouldn't have hardcoded urls in these files IMO, they should be extracted to an .env file or some file where all urls are. Also being able to configure against the real `stats.webb.tools` is what we eventually want so to have it be configurable is best here @vutuanlinh2k2

Reply:
Let's put them here `libs/webb-ui-components/src/constants/index.ts` @vutuanlinh2k2. I also found some common links in the Tangle website like the docs link, stats link, and bridge link that we can move into the `constants` file in the `webb-ui-components`  for the sharable purposes between the apps.